code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -910,6 +910,31 @@
 	required_reagents = list(SOLANINE = 1, WATER = 4)
 	result_amount = 5
 
+// Special Reactions for Plasma Beaker
+/datum/chemical_reaction/plasmabeakerdexalin
+	name = "Plasma Beaker Dexalin"
```

Comment:
I feel like this is messier than just making them subtypes of the recipes (right under the relevant recipes) and overriding the two vars as opposed to copypasting all of them

Reply:
That seems valid but it's consistent with how it's done everywhere else. e.g.: just above here, you can see the botany reactions.